wfe2/wfe.go

```diff
@@ -2336,9 +2338,10 @@ func (wfe *WebFrontEndImpl) NewOrder(
 		// if the order is a renewal, and thus exempt from the NewOrdersPerAccount
 		// and CertificatesPerDomain limits.
 		timestamps, err := wfe.sa.FQDNSetTimestampsForWindow(ctx, &sapb.CountFQDNSetsRequest{
-			DnsNames: names,
-			Window:   durationpb.New(120 * 24 * time.Hour),
-			Limit:    1,
+			DnsNames:    names,
+			Identifiers: pbIdents,
```

Comment:
Similar to comment above, we don't want to be carrying around both `names` and `pbIdents`, in case some bug causes them to diverge. These should just be
```suggestion
			DnsNames:    idents.ToDNSSlice(),
			Identifiers: idents.ToProto(),
```

Reply:
Getting rid of `pbIdents`, but same response/question as above for `names`.